apptail/cmd/apptail/main.go

```diff
@@ -1,24 +1,44 @@
 package main
 
 import (
+	"flag"
+	"fmt"
 	"github.com/ActiveState/log"
 	"github.com/ActiveState/logyard-apps/apptail"
 	"github.com/ActiveState/logyard-apps/apptail/docker"
 	apptail_event "github.com/ActiveState/logyard-apps/apptail/event"
+	"github.com/ActiveState/logyard-apps/apptail/storage"
 	"github.com/ActiveState/logyard-apps/common"
 	"github.com/ActiveState/stackato-go/server"
 	"github.com/alecthomas/gozmq"
-	"github.com/nu7hatch/gouuid"
+	uuid "github.com/nu7hatch/gouuid"
 	"io/ioutil"
 	"os"
+	"time"
+)
+
+var (
+	gob_path = flag.String("gob_path", default_path, "Path to the gob file")
```

Comment:
Can we refer to this as a apptail's "state file" externally? "Gob" is just an implementation detail.